server/test/auth/ApiKeyGrantsTest.java

```diff
@@ -0,0 +1,66 @@
+package auth;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class ApiKeyGrantsTest {
+
+  private static final String PROGRAM_1_SLUG = "program-1";
+  private static final String PROGRAM_2_SLUG = "program-2";
+  private ApiKeyGrants subject;
```

Comment:
question: where is the use of "subject" in our tests from? I haven't seen it as a generic identifier in this context and it was somewhat unreadable to me in other test code.  I've only seen cut (Code under Test).

Reply:
changed to `apiKeyGrants`.

unit tests where the object under test is referred to as `subject` is an old habit, I agree it's not a very clear naming convention.